enterprise_subsidy/apps/transaction/management/commands/write_reversals_from_enterprise_unenrollments.py

```diff
@@ -44,94 +42,20 @@ def add_arguments(self, parser):
             ),
         )
 
-    def handle_reversing_enterprise_course_unenrollment(self, unenrollment):
```

Comment:
All the logic in this method was duplicative of the same logic in handlers.py.  The duplicate code was combined, and adapted to work for both the signal handler and this management command.